### ISSUE TITLE
Output file name issues

### ISSUE THREAD

**Issue by gino0631** [adrobisch/raml-converter#6]
Maven plugin of the RAML converter determines the name of the output file by substring'ing the original RAML file name by the last index of File.separatorChar (RamlHtmlMojo.java:68), so this puts a requirement for the file name to have platform-specific separators. As this is not necessarily true (it is common to have a file name with forward slashes like `../../src/raml/example.raml` in a pom.xml regardless of a platform), this logic fails if the platform uses a different separator (e.g. `\` on Windows). I'd suggest using `File.getName()` instead.

BTW, the name of the output file produced this way has a double extension - `.raml.html`. In my opinion, it would be more natural to replace the extension (like .java -> .class, not .java.class). If this is by design, OK, but it would be good to have an option to override this behaviour by providing the output name explicitly in this case.

**Comment by adrobisch** [comment]
This should be improved with https://github.com/adrobisch/raml-converter/commit/c2be68374d0640083166f20cbea59b5921a42fe8.

I agree that the output filename should be configurable. Maybe the best would be to specify some kind of pattern, since we allow multiple RAML input files.

**Comment by gino0631** [comment]
Yes, the first part (regarding platform-specific separators) is resolved.

As for the filenames, I don't think we need complicated renaming rules or patterns - other plugins could be used for this purpose, if necessary. The question is only about the extensions of generated files. I don't know what the practice is, but, for instance, examples of [raml2html](https://github.com/raml2html/raml2html#as-a-command-line-script) suggest just `.html`. If there are other reasons for the present design, or the change seems to break something, I'll be happy to leave it as-is. At a maximum, the solution could be to introduce a switch to replace original extensions.
